python/paddle/fluid/tests/unittests/test_fetch_list.py

```diff
@@ -0,0 +1,70 @@
+#  Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
```

Comment:
This filename is too simple to know what it works for. Please rename the filename. By the way, should we add unittests independent of op_test, which is only used to test whether `exe.run(return_numpy=False) is right`?

Reply:
Done, thanks!